package.json

```
@@ -4,12 +4,20 @@
   "scripts": {
     "bootstrap": "lerna bootstrap",
     "clean": "lerna clean --yes",
-    "build:docs": "yarn workspace cra run build:storybook",
+    "compile:core": "yarn workspace @evilfactory/anggun-core compile",
+    "compile:core:watch": "yarn workspace @evilfactory/anggun-core compile:watch",
+    "build:docs": "yarn workspace cra run build",
```

```suggestion
    "build:docs": "lerna run build",
```
when use this is suggestion `without scope`, this is mean you run `build` for all packages.

---

btw, if I use this command, I can't see the log

I prefer use the previous one @ri7nz

---

@kevanantha add options ```--stream``` or ```--paralel```

---

noted, thanks